apps/files/models.py

```diff
@@ -80,6 +80,8 @@ class File(amo.models.OnChangeMixin, amo.models.ModelBase):
     binary_components = models.BooleanField(default=False, db_index=True)
     # Serial number of the certificate use for the signature.
     cert_serial_num = models.CharField(max_length=255, blank=True)
+    # Is the file signed by Mozilla?
+    is_signed = models.BooleanField(default=False)
```

Comment:
The `.1-signed` hackery is only to distinguish (and bump) versions that have been manually signed: we'll only be using that for the roll-out of the signing feature. We won't use it anymore afterwards.

Regarding the multiple files per versions and some of those not signed: yes, this is possible in at least two different cases:
1/ you create a new version for the linux platform, and put it through review. It gets reviewed, and thus the file for linux is signed. You can now add another file (to the same version) for the mac platform, which isn't signed until it'll be reviewed
2/ when a version with multiple files is reviewed, each individual file is signed in turn. If something happens with the signature of a file, it'll raise an exception and stop the process (and thus not sign the remaining files).